playwright/e2e/12-notification-modal.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { expect } from '@playwright/test'
+
+import { test } from '../fixtures'
+test.beforeEach(async ({ layoutPage, loginPage }) => {
+  await layoutPage.gotoHome('dashboard')
+  await loginPage.loginDashboard()
+})
+test.describe('12. TEST Notification Modal ', () => {
```

Comment:
Add a single line space between logical code blocks for better readability.

```suggestion
import { test } from '../fixtures'

test.beforeEach(async ({ layoutPage, loginPage }) => {
  await layoutPage.gotoHome('dashboard')
  await loginPage.loginDashboard()
})

test.describe('12. TEST Notification Modal ', () => {
```